sdk/eventgrid/azure-eventgrid/azure/eventgrid/_models.py

```diff
@@ -176,14 +175,14 @@ def model(self):
     
     def _deserialize_data(self, event_type):
         """
-        Sets self._model.data to strongly typed event object if event type exists in _event_type_mappings.
+        Sets self._model.data to strongly typed event object if event type exists in _event_mappings.
         Otherwise, sets self._model.data to None.
 
         :param str event_type: The event_type of the EventGridEvent object or the type of the CloudEvent object.
         """
         # if system event type defined, set model.data to system event object
-        if event_type in DeserializedEvent._event_type_mappings:
-            self._model.data = (DeserializedEvent._event_type_mappings[event_type]).deserialize(self._model.data)
+        if event_type in _event_mappings:
```

Comment:
God I feel VERY nitpicky saying this, but it's like the most "python idiom" thing ever; but try/except on keyError?  (One could feasibly make an argument the explicit if is more comprehensible but, ehh~.  I'm not gonna bikeshed on this more though so if you feel strongly etc.)

Reply:
changed to try except - agree it's pythonic